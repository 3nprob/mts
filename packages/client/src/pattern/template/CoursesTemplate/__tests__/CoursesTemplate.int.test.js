import React from 'react'
import { defaultRender } from './CoursesTemplate.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<CoursesTemplate />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
