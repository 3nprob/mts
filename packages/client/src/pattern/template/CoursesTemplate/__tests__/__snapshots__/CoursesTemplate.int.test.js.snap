// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoursesTemplate /> Snaphot must match defaultRender 1`] = `
<div
  className="coursesTemplate"
>
  CoursesTemplate under construction.
</div>
`;
